merkletree/tb.go

```diff
@@ -1,6 +1,7 @@
 package merkletree
 
 type TemporaryBinding struct {
+	str   []byte
```

Comment:
I'm wondering if we need to include a field for the STR signature in the TB if the STR struct also exposes the signature, and the server needs to return the STR to the client for verification anyway?